gcc-8.intrinsics.imp

```diff
@@ -0,0 +1,61 @@
+# based on the #error pragmas in the header files below
```

Comment:
Can probably collapse these two lines into one sentence, e.g "These mappings are based on #error pragmas in the header files below in gcc 8.1.0."